w5e2.py

```diff
@@ -0,0 +1,37 @@
+#Cup of join
+''' Write a function called join, which receives an unlimited number of lists, each list as a parameter.
+ The function must be able to have another parameter called september.
+ The function must return one list consisting of all the lists received as parameters.
+ If the sep parameter is provided, it should be threaded as an element between any two lists. If not provided, the character "-" should be threaded in place.
+'''
+def join(*args, sep='-'):
```

Comment:
pick a better name then args

Reply:
def join(*list_of_lists, separator='-'):

Reply:
dont comment here, change the code